pkg/generator/templates.go

```diff
@@ -529,14 +551,19 @@ spec:
       containers:
         - name: {{.ProjectName}}
           image: {{.Image}}
+          ports:
+          - containerPort: {{.MetricsPort}}
+            name: metrics
```

Comment:
The string `metrics` is being used here and in `InitOperatorService()`.
We should make this a constant in k8sutil/constants.go.
And then pass it as a parameter `name: {{.MetricsPortName}}`.